schemas/json/block.json

```
@@ -144,6 +144,18 @@
 								}
 							]
 						},
+						"format": {
+							"description": "The format provides additional information about the attribute's shape.",
+							"type": "string",
+							"enum": [
+								"date",
+								"datetime",
+								"time",
+								"uri",
+								"url",
+								"email"
+							]
```

Here's an alternative [list of formats already used by WordPress](https://developer.wordpress.org/rest-api/extending-the-rest-api/schema/#format) that we could use here.

```suggestion
							"enum": [
								"date-time",
								"uri",
								"email",
								"ip",
								"uuid",
								"hex-color"
							]
```

---

That’s probably better idea to reuse what REST API supports.

New Abilities API uses exactly the same schema as REST API to reuse code, but more importantly to avoid confusion for developers when writing these schemas.